store/auth.go

```diff
@@ -158,3 +160,41 @@ func ReadStoreToken() (*StoreToken, error) {
 
 	return &readStoreToken, nil
 }
+
+// RequestPackageAccessMacaroon requests a macaroon for accessing package data from the ubuntu store
+func RequestPackageAccessMacaroon() (string, error) {
+	const errorPrefix = "cannot get package access macaroon from store: "
+
+	emptyJSONData := "{}"
+	req, err := http.NewRequest("POST", myappsPackageAccessAPI, strings.NewReader(emptyJSONData))
+	req.Header.Set("accept", "application/json")
+	req.Header.Set("content-type", "application/json")
```

Comment:
these two need to go after the error check, usually if err != nil the other return argument are nil so this would panic